packages/image_picker/image_picker_platform_interface/test/new_method_channel_image_picker_test.dart

```diff
@@ -326,8 +326,8 @@ void main() {
         // ignore: deprecated_member_use_from_same_package
         final LostData response = await picker.retrieveLostData();
         expect(response.type, RetrieveType.video);
-        expect(response.exception.code, 'test_error_code');
-        expect(response.exception.message, 'test_error_message');
+        expect(response.exception?.code, 'test_error_code');
```

Comment:
nit: check that `response.exception` is not `null`, and then `response.exception!.code`